### PR TITLE
[WIP] A 'Friends of PHP' alternative to Code Sniffer

### PR DESCRIPTION
**Pull request by niels-nijens**
This PR adds a suggestion to change the PHP code style 'linter' to the PHP CS fixer with the Symfony code standards configured.

### CODE REVIEW

File: SuperBraveGdprBundle.php [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
  *
  * @category  Bundle
- * @package   Gdpr
+ *
  * @author    Superbrave <info@superbrave.nl>
  * @copyright 2018 Superbrave <info@superbrave.nl>
  * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
- * @link      https://www.superbrave.nl/
+ *
+ * @see      https://www.superbrave.nl/
```
Comment (hansgoed) [inline SuperBraveGdprBundle.php]: Shouldn't this be the github page?

**Comment by stokjes** [comment]
By using the cs-fixer instead of the current code sniffer, we can lose the phpcs.xml file right?

**Comment by niels-nijens** [comment]
Yes, we can. I can remove the configuration file and dependency in this PR when we decide we want to use the PHP CS fixer instead of the code sniffer.

**Comment by Garrcomm** [comment]
Why "instead" and not just use both checks? I know the PSR2 standard doesn't validate docblocks. Also code sniffer comes with a patcher tool to fix most changes.

**Comment by kiettran** [comment]
This pull request does no longer need to be requested. It already has been done in the latest [pull request](https://github.com/superbrave/gdpr-bundle/pull/38).

**Pull request closed without merging**